frontend/e2e/annotation-mode-page/annotation-mode.page.spec.ts

```diff
@@ -221,6 +244,62 @@ test.describe("Annotate page", () => {
 });
 
 test.describe("Annotation page shortcuts", () => {
+  test.describe("Global shortcuts", () => {
+    test("when focus is not in form, press Tab to focus on first question", async ({ page }) => {
+      await goToAnnotationPage(page);
+      const formBox = await page.getByText("Submit your feedback").boundingBox();
+
+      await page.mouse.click(formBox.x - 25, formBox.y - 25); // click outside form to loose focus
+      await expect(page).toHaveScreenshot();
+
+      await page.keyboard.press(shortcuts.focusOnForm);
+      await expect(page).toHaveScreenshot();
+    });
+    test("on first record, go to next record by pressing Ctrl+ArrowRight", async ({ page }) => {
+      await goToAnnotationPage(page);
+      const formBox = await page.getByText("Submit your feedback").boundingBox();
+
+      await page.mouse.click(formBox.x - 25, formBox.y - 25); // click outside form to loose focus
+      await expect(page).toHaveScreenshot();
+
+      await page.keyboard.press(shortcuts.goToNextRecord);
+      await expect(page).toHaveScreenshot();
+    });
+    test("on first record, can't go to prev record by pressing Ctrl+ArrowLeft", async ({ page }) => {
+      await goToAnnotationPage(page);
+      const formBox = await page.getByText("Submit your feedback").boundingBox();
+
+      await page.mouse.click(formBox.x - 25, formBox.y - 25); // click outside form to loose focus
+      await expect(page).toHaveScreenshot();
+
+      await page.keyboard.press(shortcuts.goToPrevRecord);
+      await expect(page).toHaveScreenshot();
+    });
+    test("on last record, can't go to next record by pressing Ctrl+ArrowRight", async ({ page }) => {
+      await goToAnnotationPage(page);
+      const formBox = await page.getByText("Submit your feedback").boundingBox();
+
+      await page.mouse.click(formBox.x - 25, formBox.y - 25); // click outside form to loose focus
```

Comment:
I think that you mixing somethings here, I think that you just want to move next record, is not necessary to click outside.

Reply:
Since by default when the user is arriving in the page, we have the focus on the form, I force the click only to loose the focus and then when pressing `tab` get the focus again in the form. 
Is there a better way than clicking outside the form to loose the focus ?